src/config.ts

```diff
@@ -1,5 +1,6 @@
 export const config = {
   app: 'Shorebird',
+  tagline: 'Build and deploy amazing multi-platform experiences with Flutter',
```

Comment:
Not "Code push for Flutter"?

Reply:
That's what it was before and I think it was too short so search engines don't pick it up 🤷

Reply:
I would lead with what we do, which right now is "code push".

e.g.
Use code push for Flutter to deliver instant updates to all devices.  Patch issues in your iOS or Android Flutter apps instantly.

But I also don't need to bike shed. :). Thanks for doing this!